src/Backend/Modules/MediaLibrary/Domain/MediaGroupMediaItem/Exception/MediaGroupMediaItemNotFound.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary\Domain\MediaItem\Exception;
+
+class MediaGroupMediaItemNotFound extends \Exception
+{
+    public static function forMediaItemId(string $id): self
+    {
+        return new self('Can\'t find a MediaGroupMediaItem with mediaItemId = "' . $id . '"".');
```

Comment:
String interpolation would look better here.

Reply:
Used InvalidArgument since it is about a string and not a specific class